101-function-app-create-dynamic-slot/azuredeploy.parameters.us.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "appInsightsLocation": {
+          "value": "usgovvirginia"
```

Comment:
Is it weird we have Gov cloud in our samples? Most customers can't run in Gov cloud, right?

Reply:
I think @bmoore-msft may have more context but both of the other Quickstart templates for Functions have this sovereign cloud support.

Reply:
All of our Gov cloud customers can run in Gov cloud ;)

We require it for samples unless the platform support has a gap.